content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -209,6 +209,7 @@ For a reference to old feature gates that are removed, please refer to
 | `TopologyManagerPolicyBetaOptions` | `true` | Beta | 1.28 | |
 | `TopologyManagerPolicyOptions` | `false` | Alpha | 1.26 | 1.27 |
 | `TopologyManagerPolicyOptions` | `true` | Beta | 1.28 | |
+| `UnauthenticatedHTTP2DOSMitigation` | `false` | Beta | 1.29 | |
```

Comment:
- Is the default value true or false?

Reply:
In 1.29 it defaults to true

It was backported to earlier releases, and in those it defaults to false

Reply:
Got it.

OK, so we _also_ need a PR against _dev-1.29_

Reply:
In the docs do we want to include that it was backported to 1.28.3?  To ensure that it is available and not only in 1.29?

Reply:
This PR is targeting the v1.28 docs; we don't need to mention that the docs are for v1.28 (that's implied, although it might be nice to make that clear on each page).

To be clear: making the version more visible would be an entirely separate feature request. My key point is that this PR is already OK in respect of making the affected version clear.